assets/scss/components/_hero.scss

```diff
@@ -0,0 +1,13 @@
+.hero {
+	
+	
+
+
+}
+
+.js-slider {
```

Comment:
This is interesting.  I wouldn't dock you for not knowing this, but the main reason we use a '.js-' class for javascript stuff is partially because you know it is controlling javascript, not styles.  That being said, I wouldn't style any of the '.js-' classes.  Use another class on the element, or add classes as needed.